modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/prepare/ddl/DdlSqlToCommandConverterTest.java

```diff
@@ -246,9 +210,95 @@ public void tableWithAutogenPkColumn() throws SqlParseException {
         );
     }
 
-    private TableOptionInfo tableOptionInfo(String name) {
-        return new TableOptionInfo<>(name, Object.class, null, (createTableCommand, o) -> {
-        });
+    @Test
```

Comment:
the same question about moving to a separate class

Reply:
Done, thanks.